tmva/sofie/inc/TMVA/SOFIE_common.hxx

```diff
@@ -105,8 +105,8 @@ ETensorType GetTemplatedType(T /*obj*/ ){
 namespace UTILITY{
 template<typename T>
 T* Unidirectional_broadcast(const T* original_data, const std::vector<size_t> original_shape, const std::vector<size_t> target_shape);
-template <typename T>
-std::vector<size_t> Multidirectional_broadcast(const T* original_data,  std::vector<size_t> input1_shape, std::vector<size_t> input2_shape);
+
+std::vector<size_t> Multidirectional_broadcast(std::vector<size_t> input1_shape, std::vector<size_t> input2_shape);
```

Comment:
One more thing, better to pass input vector as const reference,
```
std::vector<size_t> Multidirectional_broadcast(const std::vector<size_t>  & input1_shape, const std::vector<size_t> & input2_shape);
```

Reply:
This gives an error while building.